src/broadcast.rs

```diff
@@ -205,7 +208,16 @@ impl TransmitQueue {
             let mut prune = vec![];
 
             for item in self.set.range(start..end) {
-                let size = serialized_size(&item.broadcast)? as i64;
+                // Drop those broadcasts which cannot be serialized.
+                let size = match serialized_size(&item.broadcast) {
+                    Ok(n) => n as i64,
+                    Err(e) => {
+                        eprintln!("Serialization error: {:?}", e);
```

Comment:
you can uncomment the `tokio-trace` stuff in `lib` and use `error!()` here.

Reply:
Oh yeah, since this is a library, we don't need to initialize anything right?

Reply:
correct